src/components/atoms/SfAlert/SfAlert.html

```diff
@@ -0,0 +1,19 @@
+<div class="sf-alert" :class="`sf-alert--${type}`">
+    <slot>
```

Comment:
this default slot should just be at the bottom, then we have additional ability to add content instead of only erasing it